File: SkyEditor.Core (VB)/Projects/ProjectAddedEventArgs.vb
Namespace Projects
    Public Class ProjectAddedEventArgs
        Inherits EventArgs

        ''' <summary>
        ''' Path of the project in the solution
        ''' </summary>
        ''' <returns></returns>
        Public Property Path As String

        ''' <summary>
        ''' The project that was added
        ''' </summary>
        ''' <returns></returns>
        Public Property Project As Project
    End Class

End Namespace

File: SkyEditor.Core/Projects/ProjectAddedEventArgs.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SkyEditor.Core.Projects
{
    public class ProjectAddedEventArgs : EventArgs
    {

        /// <summary>
        /// Path of the project in the solution
        /// </summary>
        public string Path { get; set; }

        /// <summary>
        /// The project that was added
        /// </summary>
        public Project Project { get; set; }
    }
}
